README.md

```diff
@@ -0,0 +1,16 @@
+### How help-activity works:
+
+when contextual help( <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>h</kbd> ) is requested from within an sugar activity, the [helplink.json](helplink.json) file matches the bundle_id(the same as defined in the particular activity.info file) with a corresponding html page and renders its content in a tiny browser.
+
+Alternatively, all documentation is also available at https://help.sugarlabs.org/
+
+### How to edit Help and Contribute:
+https://help.sugarlabs.org/en/how_to_help.html
+
```

Comment:
Probably is better link to the link in this repository https://github.com/godiard/help-activity/blob/master/source/how_to_help.rst

Reply:
changed.